### PR TITLE
fix minor typo and wording in troubleshooting.rst

### PR DESCRIPTION
**Pull request by flagbrad**
Thanks for making a pull request to converse.js!

Before submitting your request, please make sure the following conditions are met:

- [ ] Add a changelog entry for your change in `CHANGES.md`
- [ ] When adding a configuration variable, please make sure to
      document it in `docs/source/configuration.rst`
- [ ] Please add a test for your change. Tests can be run in the commandline
      with `make check` or you can run them in the browser by running `make serve`
      and then opening `http://localhost:8000/tests.html`.

### CODE REVIEW

File: docs/source/troubleshooting.rst [hunk 1]
```diff
@@ -147,9 +147,9 @@ Use the Web Auth API
 ********************
 
 Converse supports the `Web Authentication API <https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API>`_
-which let's it use the secure credential management of the browser to get the
-uesr credentials to automatically log the user in. This however requires that
-the user saves his or her username and password in the browser. Often the user
+which uses the secure credential management of the browser to get the
+uesr credentials to automatically log the user in. This requires
```
Comment (jcbrand) [inline docs/source/troubleshooting.rst:151]: Thanks. `user` is still misspelled as `uesr`, can you please fix that as well?

Comment (flagbrad) [reply docs/source/troubleshooting.rst]: Done, thanks @jcbrand . Also did some light copy editing for readability if that seems okay.

File: docs/source/troubleshooting.rst [hunk 2]
```diff
@@ -74,11 +70,12 @@ Chrome <https://developer.chrome.com/devtools/docs/cpu-profiling>`_ to find
 bottlenecks in the code.
 
 However, with large rosters (more than 1000 contacts), rendering in
-Converse slows down a lot and it may become intolerably slow.
+Converse is known to slow down considerbly. It may become intolerably slow
```
Comment (jcbrand) [inline docs/source/troubleshooting.rst:73]: Typo: considerably

File: docs/source/troubleshooting.rst (hunk 1 above)
Comment (jcbrand) [reply docs/source/troubleshooting.rst]: Definitely, thanks, I just found one typo.

File: docs/source/troubleshooting.rst (hunk 2 above)
Comment (flagbrad) [reply docs/source/troubleshooting.rst]: @jcbrand Thanks, fixed a few more typos. Thanks for all your work, this is a terrific XMPP client.

**Pull request merged**